README.md

```diff
@@ -19,7 +19,21 @@ After downloading the module in the modules folder, visit the Extend (/admin/mod
 
 ## Configuration
 
-@ToDo
+1) Enable the module
+2) visit /media/add to add media content to website.
+3) Visit /admin/structure/media to edit the default media bundles.
+
+### Using Slick to modify gallery display
+
+You can use the slick library to improve the gallery display.
+
+1) Download [Slick Library](https://github.com/kenwheeler/slick/) and extract it in libraries directory. Required assets should be available at /libraries/slick/slick/slick.css
```

Comment:
Okay. I'll mention the required libraries as a requirement for the module. And should we include the slick extras module too or mention in the readme for extra styles. https://www.drupal.org/project/slick_extras